sdc/tests/tests_perf/test_perf_df.py

```diff
@@ -97,9 +60,8 @@ def _test_python(self, pyfunc, record, *args, **kwargs):
         record["test_results"], _ = \
             get_times(pyfunc, *args, **kwargs)
 
-    def _test_case(self, pyfunc, data_name, total_data_length, test_name=None,
-                   input_data=test_global_input_data_float64):
-        test_name = test_name or data_name
+    def _test_case(self, pyfunc, data_name, total_data_length, input_data=test_global_input_data_float64):
+        test_name = 'DataFrame.{}'.format(data_name)
```

Comment:
I see that `data_name` and DataFrame function are the same. I think it is possible to have many data with different name for testing one function. So `test_name` is `DataFrame.data_for_min` - is not it strange?

Reply:
Explain otherwise please.

Reply:
There is 2 different peaces of data - test name and data used for this test. Now test name depends on data name.
Why did `test_name` parameter was deleted?
How to test e.g. function min() with floats and int data? How to test min with different parameters?